backend/src/main/java/com/huellapositiva/application/controller/ProposalApiController.java

```diff
@@ -437,6 +437,10 @@ public ProposalResponseDto fetchProposalWithVolunteers(@PathVariable String idPr
                             responseCode = "204",
                             description = "No Content, proposal status changed to CANCELLED successfully."
                     ),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "Bad Request, illegal status."
+                    ),
```

Comment:
Bar request != illegal status

That the proposal cannot be cancelled does not mean the request is malformed. The request was indeed valid so therefore we should return a more suitable status code like 403, 409, 412

Reply:
We changed it to 412.